CHANGELOG.md

```diff
@@ -1,158 +0,0 @@
-# Changelog
```

Comment:
Its ok to delete this? or should i restore it before merge?

Reply:
Reviewed this in meet with @carlosescri ,we will restore this file before merge and include last changes that we have done